camilo-escobar/tarea-3/whiteOrBlack.js

```diff
@@ -0,0 +1,10 @@
+function mineColor(file, rank) {
+  const files = ["a", "b", "c", "d", "e", "f", "g", "h"];
+  const ranks = [1, 2, 3, 4, 5, 6, 7, 8];
+  return Even(files.indexOf(file)) === Even(ranks.indexOf(rank))
+    ? "black"
+    : "white";
+}
+function Even(number) {
```

Comment:
@camiloesag , this is the exact same solution from other student, please find a a way to make it a one line solution